src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -32,30 +31,24 @@ public Configuration(File reportOutputDirectory, String projectName) {
         this.projectName = projectName;
     }
 
-    public void setStatusFlags(boolean failsIfSkipped, boolean failsIFPending, boolean failsIfUndefined,
-            boolean failsIfMissing) {
+    public void setStatusFlags(boolean failsIfSkipped, boolean failsIfPending, boolean failsIfUndefined) {
         this.failsIfSkipped = failsIfSkipped;
-        this.failsIFPending = failsIFPending;
+        this.failsIfPending = failsIfPending;
```

Comment:
this will require plugin update - can I ask you to prepare PR for the plugin so I will merge it with next release?